packages/opencensus-nodejs/src/trace/tracing.ts

```diff
@@ -33,12 +32,12 @@ export class Tracing implements core.Tracing {
   private defaultPlugins: core.PluginNames;
   /** A configuration object to start the tracing */
   private configLocal: core.Config = {};
-  /** An object to log information to */
-  private logger: core.Logger = null;
+  /** An object to log information to. Logs to the JS console by default. */
+  private logger: core.Logger = console;
```

Comment:
Optional: I believe there is also a logger instance in https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/common/console-logger.ts

That does formatting a bit differently from the default `console` instance I believe. I used `console` in OC Web because I didn't want to introduce extra bytes of code for the logger, but that's not really much of an issue for Node.

Reply:
Makes sense, done.